tasks/lab.js

```diff
@@ -46,7 +46,7 @@ module.exports = function (grunt) {
 		_.forIn(config, function (configValue, configName) {
 			var option = _.first(_.where(labOptions, { name : configName }));
 
-			if (option && configValue) {
+			if (option && (configValue || typeof configValue === "number")) {
```

Comment:
Using the lodash [`_.isNumber`](https://lodash.com/docs#isNumber) function would be ideal. It's the same thing, but lodash is used everywhere else.